docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
+----------------
+
+.. _collection-list:
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+This endpoint allows you to list all collections authored by this user.
+Access is limited to your own collections, or users with `Users:Edit` permission.
+
+.. http:get:: /api/v3/accounts/account/(int:user_id|string:username)/collection/
+
+    :>json int count: The number of results for this query.
+    :>json string next: The URL of the next page of results.
+    :>json string previous: The URL of the previous page of results.
+    :>json array results: An array of :ref:`collections <collection-detail-object>`.
+
+
+-----------------
+Collection Detail
+-----------------
+
+.. _collection-detail:
+
+This endpoint allows you to fetch a single collection by its `slug`.
+Access to non-`listed` collections are limited to the author, or users with `Users:Edit` permission
```

Comment:
It's unclear what 'users with `Users:Edit` permission' refers to (like, which user?) so I think it would make sense to stick with second-person voice as you did up above. Here is an example of a rewrite:

It returns any `listed` collection by the specified user. You can access a non-`listed` collection only if it was authored by you, the authenticated user. If your account has the `Users:Edit` permission then you can access any collection.